src/manifest/mod.ts

```diff
@@ -52,10 +54,6 @@ export class SlackManifest {
         bot_user: {
           display_name: def.displayName || def.name,
         },
-        app_home: {
```

Comment:
This auto-defaults remote apps to these values as well unless explicitly set by the dev which seems unintentional. I have moved the logic setting these defaults to to ROSI specific `assignRunOnSlackManifestProperties`.